tests/test_e2e.py

```diff
@@ -14,40 +14,38 @@ def wait_for_task(client: FlaskClient, name: str) -> None:
                           content_type='application/json')
         assert resp.status == '200 OK'
         status = json.loads(resp.data)['status']
-        print(status)
         assert 'Failed' not in status
         if 'Done' in status:
             break
 
 
 def test_e2e(app: Flask, fs: FakeFilesystem) -> None:
-    seq_pattern = (
-        'react more swiftly to comply with international instruments %d')
+    seq_pattern = ('Continue working for the eradiction of poverty %d')
     instance_path = './testdata/test_model/test_instance'
     fs.add_real_directory(instance_path)
-    fs.remove_object('./testdata/test_model/test_instance/quality.json')
     fs.remove_object('./testdata/test_model/test_instance/thresholds.json')
     client = app.test_client()
     with app.test_request_context():
-        # initial classify returns no topics because we deleted thresholds.json.
+        # without threshold file default quality is set to 0.5
```

Comment:
for all topics?

Reply:
It's difficult to determine the quality of a prediction if we don't have data to optimize thresholds and evaluate performance. I can be super good, but also quite bad - so I hoped that with a default confidence 0.5 we could display this arbitrary decision...
Still I do see a point that for end users 0.5 might seem quite as a quite confident decision. I lowered it down to 0.3